_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,103 @@ Parameter | Description | Type | Required
+                                "target_index": "target",
+                                "page_size": 1000,
+                                "dimensions": [
+                                    {
+                                        "date_histogram": {
+                                            "fixed_interval": "60m",
+                                            "source_field": "order_date",
+                                            "target_field": "order_date",
+                                            "timezone": "America/Los_Angeles"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "customer_gender",
+                                            "target_field": "customer_gender"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "day_of_week",
+                                            "target_field": "day_of_week"
+                                        }
+                                    }
+                                ],
+                                "metrics": [
+                                    {
+                                        "source_field": "taxless_total_price",
+                                        "metrics": [
+                                            {
+                                                "sum": {}
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "source_field": "total_quantity",
+                                        "metrics": [
+                                            {
+                                                "avg": {}
+                                            },
+                                            {
+                                                "max": {}
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                ],
+                "transitions": []
+            }
+        ]
+    }
+}
+````
+
+#### Field parameters
+
+The following field parameters are required:
+
+Field Parameter | Data Type | Description 
+:--- | :--- |:---
+source_field | String | The field(s) to transform.
+target_field | String | The target field(s) the newly transformed data is added to.
+fixed_interval | String | Define the interval period. Specify the fixed interval in milliseconds, seconds, minutes, hours, or days.
```

Comment:
The parameter type should be "Time" here. This is a special human-readable time. We can also add examples like `60m`.